Automattic/wp-calypso
# Featured image email toggle: Remove toggle from Writing settings page

#### Proposed Changes

* remove the Featured image email template toggle from the Writing settings page - as it is already available in the new Reading settings page; Fixes #71132.

|  Before | After |
| ------------- | ------------- |
| ![Markup on 2022-12-19 at 11:41:34](https://user-images.githubusercontent.com/25105483/208407950-48bb48ba-f1c4-42ab-9c8a-6d77e2367f82.png) | ![Markup on 2022-12-19 at 11:42:03](https://user-images.githubusercontent.com/25105483/208407955-9f26e29d-2e9c-47d4-a875-54c24d094820.png) |

Related project threads:

* Featured Image Template Toggle (pdDOJh-Nu-p2)
* Modernize Reading Settings (pdDOJh-17o-p2)

#### Testing Instructions



1. Navigate to _Settings → Writing_. The Featured image toggle **should not** be visible.
2. Navigate to `http://calypso.localhost:3000/settings/reading/<site-address>`. The toggle there should still be present.
3. There should be no unexpected React errors/warnings.

ℹ️ Please note that if you would like to test the actual functionality of the toggle in the Reading page, the latest Jetpack `trunk` build needs to synced to your sandbox (PCYsg-eg0-p2) with `public-api` sandboxed. This is because the latest Jetpack `trunk` build hasn't been deployed to WPCOM.

#### Pre-merge Checklist



- [x] Has the general commit checklist been followed? ('-hS-p2)


Related to #71132.
